src/averageAges.js

```diff
@@ -14,12 +40,13 @@
  *
  * @return {number}
  */
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const normalizedPeople = (century)
+    ? people.filter(person => Math.ceil(person.died / 100) === century)
+    : people;
+
+  return getAverageAge(getPeopleBySex(normalizedPeople, 'm'));
```

Comment:
Here you call `filter` 2 times, let's improve this solution and use `filter` only once?